src/instructions/Toggle.jsx

```diff
@@ -0,0 +1,23 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+
+export default function Toggle({ children }) {
+  const [showContent, setShowContent] = useState(false);
+
+  const toggleContent = () => {
+    setShowContent(!showContent);
+  };
+
+  return (
+    <div>
+      <button className="button" onClick={toggleContent}>
+        Instructions
+      </button>
+      {showContent && children}
+    </div>
+  );
+}
+
+Toggle.propTypes = {
+  children: PropTypes.node.isRequired,
```

Comment:
So this component is called Toggle which explains the behavior (to toggle something) but it is not generic because the Instructions text is explicitly displayed in the component. So perhaps this is named wrong. Maybe you have it flipped, the Instructions component toggles the instructions based on whether we should show content. 
Take a step back and see how this component is used in the parent. Does this name you used still make sense?